website/docs/fr-FR/switch.md

```diff
@@ -145,7 +145,7 @@ Switch est utilisé pour choisir entre deux états opposés.
 
 | Attribut      | Description          | Type      | Valeurs acceptées       | Défaut |
 | ----| ----| ----| ----|---- |
-| value / v-model | La valeur liée. | boolean / string / number | — | — |
+| value / v-model | La valeur liée. Elle doit être équivalente à une `active-value` ou une `inactive-value`, par défaut c'est un type `boolean`. | boolean / string / number | — | — |
```

Comment:
```suggestion
| value / v-model | Valeur liée. Elle doit être équivalente à `active-value` ou `inactive-value`, par défaut elle est de type `boolean`. | boolean / string / number | — | — |
```
Here is some improvement to the translation. Happy to help for French translation

Reply:
Thank you! cc @XingXiaoWu